backend/users/test_views.py

```diff
@@ -57,3 +67,21 @@ def test_existing_token(self):
         self.assertEqual(response.status_code, 200)
         self.assertIn("token", data)
         self.assertNotEqual(token.key, data["token"])
```

Comment:
Maybe a little much, but would adding

```
tokens = Token.objects.filter(user=user)
self.assertEqual(tokens.count(), 1)
```

or something similar here make sense?

Reply:
makes sense to me